graphbolt/src/shared_memory_utils.cc

```diff
@@ -0,0 +1,146 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ *
+ * @file shared_memory_utils.cc
+ * @brief Share memory utility function implementation.
+ */
+#include "./shared_memory_utils.h"
+
+#include <graphbolt/serialize.h>
+#include <graphbolt/shared_memory.h>
+
+namespace graphbolt {
+namespace sampling {
+
+static SharedMemoryPtr CopyTorchArchiveToSharedMemory(
+    const std::string& shared_memory_name, int64_t memory_size,
```

Comment:
same here, memory_size -> size, since under this scope, it is clear that the size is archive memory size.

Reply:
but in CopyTensorsToSharedMemory, it makes sense to use meta_memory_size to clarify.